lib/es5/utils/dag-node.js

```diff
@@ -37,7 +37,7 @@ var stringifyCid = function stringifyCid(cid) {
     return cid.map(stringifyCid);
   }
 
-  return cid.toBaseEncodedString();
+  return cid.toString();
```

Comment:
Don't we need to pass an encoding in here, i.e.

```javascript
const { base58btc } = require('multiformats/bases/base58')
cid.toString(base58btc)
```

Reply:
I would agree. But I'm unsure what Version of CID is used here.

Is it the currently maintained `multiformats` or the deprecated `cids` library?
Because this line suggests the later: 
```js
var CID = require('cids');
```
([line 9](https://github.com/orbitdb/ipfs-log/blob/56ef6b8f857986fdd6c34605d45af8ea24e698b2/lib/es5/utils/dag-node.js#L9))
https://github.com/orbitdb/ipfs-log/blob/56ef6b8f857986fdd6c34605d45af8ea24e698b2/lib/es5/utils/dag-node.js#L9

Reply:
since we are changing this to use .toString() we are assuming the new `multiformats` cid is being used.